src/components/shared/user-courses/share/progress-status.scss

```diff
@@ -0,0 +1,17 @@
+$progress-bar-color: #000;
+
+.progress-bar-wrapper {
+    position: relative;
+    width: 280px;
+
+    .progress-bar-label {
+        position: absolute;
+        left: 0;
+        top: 0;
+        font-size: .75rem;
+        margin-left: 8px;
+        color: #000;
+    }
+}
+
+@import '~@edx/paragon/scss/core/core.scss';
```

Comment:
Remove this line, we don't need to import it, we've just imported it once in the index.scss